designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,18 @@
+# Beyond basic messaging: Designing long-running processes
+
+## Abstract
+
+Messaging in distributed systems offers a range of valuable benefits: it enables the design of decoupled components, where each component can evolve and scale individually. In addition, messaging improves the system’s fault tolerance but goes even further by opening the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s exactly the focus of this workshop.
+
+If you’ve been exposed to the basic building blocks of messaging and want to understand how to design, build, and test long-running processes in message-based distributed systems, this workshop is for you.
+
+Throughout this workshop, you’ll learn:
+
+- How to identify long-running processes in your domain
+- Effective techniques to help design complex business processes
+- What orchestration and choreography is, but more importantly, when and why to use each pattern
+- How to deal with actions that occur at different points in time
+- How to deal with out-of-order events seamlessly
+- How to test workflows and troubleshoot issues where it matters most: in production!
+
+To get the most out of this workshop, some basic messaging knowledge is expected, including understanding message brokers and messaging patterns like Request-Reply and Publish-Subscribe. Get ready to dive into design exercises in small groups and unlock the secrets of messaging's real-world superpowers!
```

Comment:
In the context of @danielmarbach comments.

My impression after reading the abstract was that there would be (some) coding involved. If that is not the case maybe that is worth making it more explicit. It would be also good to say that experience in any messaging technology will do.

Reply:
I believe I'm not being explicit because I want to leave the door open. However, I think you're both right. Maybe I should make explicit that:
- They should mostly expect design exercises in group and no coding during the workshop
- They can expect samples/follow-up coding exercises

If I do it that way, and I end up having more time than I expect, I can always use those artifacts to fill up remaining time.